### PR TITLE
Debug UI tweaks

### PR DESCRIPTION
**Pull request by itowlson**
Various tweaks to debugging UI:

* Added a prompt and a hint to the 'enter debug command' input box
* Added information message when remote debug session is established
* Added the option to expose the debug deployment as a service (enables debugging of Web servers)
* Added a 'Remove Debug' command to delete the debug deployment (and service if exposed)

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -909,6 +941,41 @@ waitForRunningPod = function (name, callback) {
...
+
+function deploymentExists(deploymentName, handler) {
+    exists('deployments', deploymentName, handler);
+}
+
+function serviceExists(serviceName, handler) {
+    exists('services', serviceName, handler);
+}
+
+function removedebugKubernetes() {
```
Comment (brendandburns) [inline extension.js]: nit `removeDebugKubernetes` (camel case)

**Comment by brendandburns** [comment]
Generally looks good to me (LGTM)

one nit.

**Comment by itowlson** [comment]
Fixed casing issue and rebased onto latest master

**Comment by brendandburns** [comment]
Thanks!

**Pull request merged**